src/services/LogFileManager/decodeUtils.ts

```
@@ -0,0 +1,157 @@
+            decoderOptions,
+        );
+
+        if (null !== extensionResult) {
+            ({decoder, matchingExtension: fileExtension, fileTypeDef} = extensionResult);
+        }
+    }
+
+    // If no decoder was found by extension, try to create one based on the file's magic number.
+    if (null === decoder) {
+        console.warn(`No decoder found for file extension "${fileExtension}". ` +
+            "Trying to match by signature.");
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)